src/language/markdown-source-code.js

```diff
@@ -137,7 +141,7 @@ function extractInlineConfigCommentsFromHTML(node) {
 
 /**
  * Markdown Source Code Object
- * @implements {IMarkdownSourceCode}
+ * @implements {TextSourceCode}
```

Comment:
```suggestion
 * @implements {TextSourceCode<{ LangOptions: MarkdownLanguageOptions; RootNode: RootNode; SyntaxElementWithLoc: Node; ConfigNode: { value: string; position: SourceLocation }; }>}
```

Could we specify the type param options here like it's done in `@eslint/json`: https://github.com/eslint/json/blob/dce4601b1a40ceaeb4b61fbcbef0170a67b73e37/src/languages/json-source-code.js#L68

Reply:
Oops, I missed that. thank you.